app/assets/javascript/components/editor/editor.js

```diff
@@ -145,14 +145,14 @@ const EditorController = class extends Controller {
 
     let message;
 
-    if (numberwords.length > 150) {
-      message = `You have ${numberwords.length - 150} words too many`;
+    if (numberwords.length > 300) {
```

Comment:
whilst its not strictly to do with your work, might be nice to make this a const MAX_WORDS = 300 and use MAX_WORDS where necessary here. tbh this JS needs a tidy tho, up to you wether you do it in this PR

Reply:
Yeahh, I agree adding a constant will be much nicer, Ill do that as part of this PR. Good spot @alexbowen